<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>谷哥番茄钟</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="3">
    <widget class="QToolButton" name="tool_btn_clock">
     <property name="font">
      <font>
       <family>Yuppy SC</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>番茄钟</string>
     </property>
     <property name="icon">
      <iconset resource="res.qrc">
       <normaloff>:/Icons/promo_button.png</normaloff>:/Icons/promo_button.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>44</width>
       <height>32</height>
      </size>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextUnderIcon</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="5">
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="lineWidth">
      <number>2</number>
     </property>
     <property name="midLineWidth">
      <number>0</number>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="page">
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0" colspan="3">
        <layout class="QGridLayout" name="gridLayout_2"/>
       </item>
       <item row="1" column="0">
        <widget class="QToolButton" name="tool_btn_pause">
         <property name="font">
          <font>
           <family>Yuppy SC</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string>暂停</string>
         </property>
         <property name="icon">
          <iconset resource="res.qrc">
           <normaloff>:/Icons/pause-button.png</normaloff>:/Icons/pause-button.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextUnderIcon</enum>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QToolButton" name="tool_btn_start">
         <property name="font">
          <font>
           <family>Yuppy SC</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string>开始</string>
         </property>
         <property name="icon">
          <iconset resource="res.qrc">
           <normaloff>:/Icons/play_button_1.png</normaloff>:/Icons/play_button_1.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextUnderIcon</enum>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QToolButton" name="tool_btn_next">
         <property name="font">
          <font>
           <family>Yuppy SC</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string>快进</string>
         </property>
         <property name="icon">
          <iconset resource="res.qrc">
           <normaloff>:/Icons/fast-forward-button.png</normaloff>:/Icons/fast-forward-button.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextUnderIcon</enum>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label_pomo_working_time">
         <property name="font">
          <font>
           <family>Yuppy SC</family>
          </font>
         </property>
         <property name="text">
          <string>番茄工作时间</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="SpinSliderBox" name="box_pomo_working_time_setting" native="true"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_pomo_rest_time">
         <property name="font">
          <font>
           <family>Yuppy SC</family>
          </font>
         </property>
         <property name="text">
          <string>番茄休息时间</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="SpinSliderBox" name="box_pomo_rest_time_setting" native="true"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_pomo_long_rest_time">
         <property name="font">
          <font>
           <family>Yuppy SC</family>
          </font>
         </property>
         <property name="text">
          <string>番茄长休息时间</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="SpinSliderBox" name="box_pomo_long_rest_time_setting" native="true"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_pomo_number">
         <property name="font">
          <font>
           <family>Yuppy SC</family>
          </font>
         </property>
         <property name="text">
          <string>番茄钟个数</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="SpinSliderBox" name="box_pomo_number_setting" native="true"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QToolButton" name="tool_btn_settings">
     <property name="font">
      <font>
       <family>Yuppy SC</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>设置</string>
     </property>
     <property name="icon">
      <iconset resource="res.qrc">
       <normaloff>:/Icons/settings_button.png</normaloff>:/Icons/settings_button.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextUnderIcon</enum>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SpinSliderBox</class>
   <extends>QWidget</extends>
   <header location="global">spinsliderbox.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
